nncf/common/pruning/mask_propagation.py

```diff
@@ -110,6 +111,10 @@ def symbolic_mask_propagation(self, prunable_layers_types: List[str],
                     assert len(input_masks) == 1
                     input_mask = input_masks[0] # type: SymbolicMask
 
+                    # find input_mask for node if multiple input masks existed
+                    if isinstance(input_mask, dict):
+                        input_mask = find_input_mask_for_node(input_mask, node)
+
```

Comment:
This will be redundant if get_input_masks will be implemented like here: https://github.com/openvinotoolkit/nncf/pull/1244#discussion_r948793387

Reply:
Fix.